WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -34,20 +36,138 @@ public partial class WalletViewModel : WalletViewModelBase
 	[AutoNotify(SetterModifier = AccessModifier.Private)] private bool _isTransactionHistoryEmpty;
 	[AutoNotify(SetterModifier = AccessModifier.Private)] private bool _isSendButtonVisible;
 
-	protected WalletViewModel(UiContext uiContext, Wallet wallet) : base(wallet)
+	[AutoNotify(SetterModifier = AccessModifier.Protected)]
+	private bool _isLoading;
+
+	[AutoNotify(SetterModifier = AccessModifier.Protected)]
+	private bool _isCoinJoining;
+
+	public WalletState WalletState => Wallet.State;
+
+	private string _title;
+
+	public Wallet Wallet { get; }
+
+	public string WalletName => Wallet.WalletName;
+
+	public bool IsLoggedIn => Wallet.IsLoggedIn;
+
+	public bool PreferPsbtWorkflow => Wallet.KeyManager.PreferPsbtWorkflow;
+
+	public override string ToString() => WalletName;
+
+	public int CompareTo(WalletViewModel? other)
```

Comment:
The same as before, I would move comparison out of the type.

Reply:
Same as above.